src/Controller/ApiV1Controller.php

```diff
@@ -0,0 +1,211 @@
+<?php
+/**
+ * Tatoeba Project, free collaborative creation of multilingual corpuses project
+ * Copyright (C) 2009-2010  HO Ngoc Phuong Trang <tranglich@gmail.com>
```

Comment:
Please either use correct info here (year, name and email) or do not include a license header at all.

Reply:
OK, will do.

Reply:
Heading changed to:
![image](https://user-images.githubusercontent.com/7943077/117713828-72ad2680-b1d6-11eb-8859-29e013b31de1.png)